src/main/java/com/rabbitmq/client/impl/recovery/AutorecoveringConnection.java

```diff
@@ -103,17 +103,13 @@ public AutorecoveringConnection(ConnectionParams params, FrameHandlerFactory f,
      */
     public void init() throws IOException, TimeoutException {
         this.delegate = this.cf.newConnection();
-        this.addAutomaticRecoveryListener();
-    }
-
-    public void start() throws IOException {
```

Comment:
Digging in code history suggests that the method was introduced around the time we wanted to move Langohr and March Hare to use recovery implemented by this client, and that's why https://github.com/rabbitmq/rabbitmq-java-client/commit/1e603d9de9d2fa18689a305546bcefcf4d3e49c7 calls out Langohr specifically.

So far it looks like it's safe to delete this method.